onnxruntime/core/providers/cpu/tensor/upsample.cc

```diff
@@ -452,7 +452,7 @@ BilinearParams SetupUpsampleBilinear(const int32_t input_height,
   p.dy1 = scale_data;
   p.dy2 = p.dy1 + output_height;
 
-  p.dx1 = p.dy1 + 2 * output_height;
+  p.dx1 = p.dy1 + 2UL * output_height;  // 2UL is to make prefast happy
```

Comment:
UL is unsigned long. long is 32-bit on Windows, the same as int. So, why it works?

Reply:
Not sure why line 447 is fine though. Both are updating pointers (and not the value pointed to). 

Do you know the correct ruleset to use?